platform-configs/protectli-vp4650.robot

```diff
@@ -178,6 +182,9 @@ ${RESET_TO_DEFAULTS_SUPPORT}=                       ${TRUE}
 ${MEMORY_PROFILE_SUPPORT}=                          ${FALSE}
 ${DEFAULT_POWER_STATE_AFTER_FAIL}=                  Powered Off
 ${ESP_SCANNING_SUPPORT}=                            ${FALSE}
+${UUID_SETTABLE}=                                   ${TRUE}
```

Comment:
Maybe DCU_UUID_SUPPORT ?

Reply:
Changed as suggested [here](https://github.com/Dasharo/open-source-firmware-validation/pull/139/commits/ad98d1323929b34a1e69ec7c1e73bc6ecd7bed42#diff-b796784c908d965f4825f2852e977faa304581bce28ead97ad9c657ff9ac1652R185) and in all other configs as of commit 719854125560f4903705bec5d7ef8641d0ee14e4